Resources/Prototypes/_Goobstation/Maps/submarine.yml

```
@@ -0,0 +1,67 @@
+  mapPath: /Maps/_Goobstation/submarine.yml
+  minPlayers: 60
+  stations:
+    Submarine:
+      stationProto: StandardNanotrasenStation
+      components:
+        - type: StationNameSetup
+          mapNameTemplate: '{0} Submarine {1}'
+          nameGenerator:
+            !type:NanotrasenNameGenerator
+            prefixCreator: 'DV'
+        #- type: StationEmergencyShuttle
```

yamllint (1.35.1)

[warning] missing starting space in comment

(comments)